edu.stanford.protege.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "edu.stanford.protege",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk8"],
+    "command": "protege",
+    "finish-args": [
+        "--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
```

Comment:
does it need network access?

Reply:
Yes, Protégé needs it for updating its plugins & resolving external ontologies.

Reply:
where are this update stored?

Reply:
The updates are stored either in the `plugins` directory or a user's `~/.Protege/plugins` directory. I tried updating, but it complained about RO filesystem, then logged that it will use a user-directory but apparently did nothing. I'll try to tweak the configuration to select an adequate XDG directory.